cogs/plant_care_commands.py

```diff
@@ -105,13 +105,18 @@ async def deleteplant(self, ctx:utils.Context, *, plant_name:str):
     @commands.command(cls=utils.Command, aliases=['rename'])
     async def renameplant(self, ctx:utils.Context, before:str, *, after:str):
         """Deletes your plant from the database"""
-
-        async with self.bot.database() as db:
-            plant_has_before_name = await db("SELECT * FROM plant_levels WHERE user_id=$1 AND LOWER(plant_name)=LOWER($2)", ctx.author.id, before)
-            if not plant_has_before_name:
-                return await ctx.send(f"You have no plants with the name `{before}`.", allowed_mentions=discord.AllowedMentions(users=False, roles=False, everyone=False))
-            await db("UPDATE plant_levels SET plant_name=$3 WHERE user_id=$1 AND LOWER(plant_name)=LOWER($2)", ctx.author.id, before, after)
-        await ctx.send("Done!~")
+                           
+        if '"' in after:
```

Comment:
There are other quotes that work for D.py - notably `'`. Full list below.

", ‘, ‚, “, „, ⹂, 「, 『, 〝, ﹁, ﹃, ＂, ｢, «, ‹, 《, 〈.